recipe/shelf.py

```diff
@@ -427,30 +432,55 @@ class Meta:
         select_from = None
         engine = None
         ingredient_order = []
+        metadata = None
 
     def __init__(self, *args, **kwargs):
-        super(Shelf, self).__init__(*args, **kwargs)
-
+        self.Meta = type(self).Meta()
         self.Meta.ingredient_order = []
         self.Meta.table = kwargs.pop('table', None)
```

Comment:
add engine in here?

```self.Meta.engine = kwargs.pop('engine', None)```

Reply:
nothing seems to actually use the Meta.engine attribute, so I would rather just delete that. thoughts?

Reply:
Lets do it. Can't remember why it was there. We'll add it back in if it was needed.